src/test/skript/tests/syntaxes/expressions/1860-ExprStringCase.sk

```diff
@@ -0,0 +1,16 @@
+test "string cases":
```

Comment:
Add some weird characters here. Emojis, for example, *should* stay same when your try to capitalize them. This also helps to test the Unicode support that I'd like to see in this expression.